code-challenges.test.js

```diff
@@ -13,34 +13,93 @@
 
-// b) Create the function that makes the test pass.
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
 
 // --------------------2) Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
 
 // a) Create a test with an expect statement using the variables provided.
+describe("onlyRemainders", () => {
+  it ("takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.", () => {
+      const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
+      // Expected output: [ 2, 0, -1, 0 ]
+      const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
+      // Expected output: [ 2, 1, -1 ]
+     // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee"]
+      expect(onlyRemainders(hodgepodge1)).toEqual([ 2, 0, -1, 0 ])
+      expect(onlyRemainders(hodgepodge2)).toEqual([ 2, 1, -1 ])
+  
+  })
+})
+// ReferenceError: onlyRemainders is not defined
+// Filter through the array to determine if the value is a number
+    // If it is put it in an array
+    // Then divide each number and return their remainder
+
+    // Create a function onlyRemainders that has one parameter: array
+const onlyRemainders = (array) => {
+  // Create a variable that will be set at the array we need to filter through the mix of numbers and strings and return only the type that are numbers.
+  let newArray = array.filter(element => typeof element === 'number') 
+  // Next, create a new variable that will set the numbers to be divided by 3 and map over the first variable applying the second variable's function to each number.
+      let divisibleNum = num => num % 3
+      return newArray.map(divisibleNum)
+
+      
+  }
+  // Test Suites: 1 passed, 1 total
+  // Tests:       2 passed, 2 total
 
-const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
-// Expected output: [ 2, 0, -1, 0 ]
-const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
-// Expected output: [ 2, 1, -1 ]
 
-// b) Create the function that makes the test pass.
 
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
+describe("allCubed", () => {
+  it ("takes in an array of numbers and returns the sum of all the numbers cubed.", () => {
+    const cubeAndSum1 = [2, 3, 4]
+    // Expected output: 99
+    const cubeAndSum2 = [0, 5, 10]
+    // Expected output: 1125
+      expect(allCubed(cubeAndSum1)).toEqual(99)
+      expect(allCubed(cubeAndSum2)).toEqual(1125)
+  
+  })
+})
+// ReferenceError: allCubed is not defined
+
 
-const cubeAndSum1 = [2, 3, 4]
-// Expected output: 99
-const cubeAndSum2 = [0, 5, 10]
-// Expected output: 1125
+// Create a function called allCubed that has one parameter: an array
+const allCubed = (array) => {
```

Comment:
Logic looks great in this section!  For a stretch, consider how you can use higher-order functions to solve this last one.